incubator/hnc/api/v1alpha2/hnc_config_conversion.go

```diff
@@ -0,0 +1,190 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+
+	"sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+// ConvertTo converts this hncconfig to the Hub version (v1alpha1).
+func (src *HNCConfiguration) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1alpha1.HNCConfiguration)
+
+	// Spec
+	sspts := src.Spec.Types
```

Comment:
sspts = "Source Spec Propagation Types"?

Maybe srcTypes - just a tiny bit longer but quite a bit more clear? The loop variable can just be `st` instead.

Reply:
sspts = "src spec types". Updated to `srcSpecTypes`.